app/code/Magento/QuoteGraphQl/Model/Resolver/MergeCarts.php

```diff
@@ -31,44 +37,93 @@ class MergeCarts implements ResolverInterface
      */
     private $cartRepository;
 
+    /**
+     * @var CustomerCartResolver
+     */
+    private $customerCartResolver;
+
+    /**
+     * @var QuoteIdToMaskedQuoteIdInterface
+     */
+    private $quoteIdToMaskedQuoteId;
+
     /**
      * @param GetCartForUser $getCartForUser
      * @param CartRepositoryInterface $cartRepository
+     * @param CustomerCartResolver $customerCartResolver
+     * @param QuoteIdToMaskedQuoteIdInterface $quoteIdToMaskedQuoteId
      */
     public function __construct(
         GetCartForUser $getCartForUser,
-        CartRepositoryInterface $cartRepository
+        CartRepositoryInterface $cartRepository,
+        CustomerCartResolver $customerCartResolver,
+        QuoteIdToMaskedQuoteIdInterface $quoteIdToMaskedQuoteId
     ) {
         $this->getCartForUser = $getCartForUser;
         $this->cartRepository = $cartRepository;
+        $this->customerCartResolver = $customerCartResolver;
```

Comment:
Hi @andrewbess can you check for null, for the new params here. We can have issues during the upgrade if not. Thanks!

Reply:
Hello @prabhuram93 
Thank you for your note.
I have fixed it.
Please check.
Thank you in advance.

Reply:
Looks good. Thanks!